WordPress/Classes/ViewRelated/Post/PostCardStatusViewModel.swift

```diff
@@ -150,7 +150,9 @@ class PostCardStatusViewModel: NSObject {
                 buttons.append(.cancelAutoUpload)
             }
 
-            if canPublish {
+            if post.autoUploadAttemptsCount.intValue >= PostAutoUploadInteractor.maxNumberOfAttempts {
```

Comment:
This can use the new `autoUploadAttemptState()`. 🙂 

```suggestion
            if autoUploadInteractor.autoUploadAttemptState(of: post) == .reachedLimit {
```

Reply:
Done in ed87599b63801490c32894eded4937ba73b02768